projects/plugins/jetpack/extensions/blocks/ai-assistant/hooks/use-suggestions-from-openai/index.js

```diff
@@ -93,6 +93,19 @@ const useSuggestionsFromOpenAI = ( {
 		setIsLoadingCategories( loading );
```

Comment:
Can you put the clean-up code in the same hook instance? I think adding the proper comments can make it simpler.

Reply:
Yes, doing it.